packages/frontend/core/.webpack/runtime-config.ts

```diff
@@ -37,6 +37,7 @@ export function getRuntimeConfig(buildFlags: BuildFlags): RuntimeConfig {
       editorFlags,
       appVersion: packageJson.version,
       editorVersion: packageJson.dependencies['@blocksuite/editor'],
+      appBuildType: buildFlags.channel,
```

Comment:
based on current implementation, this should simply be `appBuildType: 'stable'` instead of reading the params.

Reply:
Beta/internal should be updated as well